web/src/js/ducks/flows.js

```diff
@@ -20,32 +23,54 @@ const defaultState = {
     view: [],
     filter: undefined,
     highlight: undefined,
+    sort: {sortColumn: undefined, sortDesc: false},
 }
 
 function makeFilterFn(filter) {
     return filter ? Filt.parse(filter) : () => true;
 }
 
+
+function makeSortFn(sort){
+    let column = _.find(flowtable_columns, c => c.name == sort.sortColumn);
```

Comment:
If we `export` all the columns in flowtable-columns.js, we can just `import columns from "../components/flowtable-columns"` here. `columns` is a `name -> function` dict then, so no need for find :smiley: